<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
 font-family:century gothic;
font-size:15px;
background-color:#ffffee;
font-style:black;
}
QFrame{
background:rgba(50,205,50,0.5);
border-radius:15px;
}
QComboBox{
border:none;
/*color:#2d3436;*/
border-bottom:1px solid #2d3436;
}
QComboBox::hover{
background: rgba(50,205,50,0.7);
}
QPushButton{
background:  rgba(50,205,50,0.7);
border-radius:20px;
border-radius:15px;
}

QPushButton:hover{
background:rgba(50,205,50,0.9);
color:white;
}

QToolButton{
background:  white;
border-radius:50px;
}
QLabel{
color: #000000;
background:transparent;
border: none;
}
QLineEdit{
background:transparent;
border:none;
border-bottom:1px solid #2d3436;
}

#tabachat{
background-color: #FFF5BD;
}
#tab_5,#tab_6{
background-color: #FFF5BD;
}
#tabvente{
background-color: #FFF5BD;
}

tabWidget::pane {
    	border-top: 5px solid #ffffee;
}

QTabBar::tab {
    background-color:#ffffee;
 	border-style: outset;
	border-width: 1px;
	border-right-color: qlineargradient(spread:pad, x1:0.4, y1:0.5, x2:0.6, y2:0.5, stop:0 rgba(115, 115, 115, 255), stop:1 rgba(62, 62, 62, 255));
	border-left-color: qlineargradient(spread:pad, x1:0.6, y1:0.5, x2:0.4, y2:0.5, stop:0 rgba(115, 115, 115, 255), stop:1 rgba(62, 62, 62, 255));
    border-bottom-color: rgb(58, 58, 58);
	border-bottom-width: 1px;
	border-top-width: 0px;
	border-style: solid;
    color: rgb(255, 153, 0);
	padding: 4px;
    font-style: #000000;
}

QTabBar::tab:selected, QTabBar::tab:hover {
   color:rgba(50,205,50,0.7);
   border-color:rgb(42, 42, 42);
   margin-left: 0px;
   margin-right: 0px;
   border-bottom-right-radius:4px;
   border-bottom-left-radius:4px;
   font-style: #000000;
}

QTabBar::tab:last:selected {
  
	border-color:rgba(50,205,50,0.7);
	margin-left: 0px;
  	margin-right: 0px;
	border-bottom-right-radius:4px;
	border-bottom-left-radius:4px;
}

QTabBar::tab:!selected {
   margin-bottom: 4px;
   border-bottom-right-radius:4px;
   border-bottom-left-radius:4px;
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Achat</string>
    </attribute>
    <widget class="QTabWidget" name="tabachat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tableView_achat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>501</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_trier">
       <property name="geometry">
        <rect>
         <x>531</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="tri">
       <property name="geometry">
        <rect>
         <x>517</x>
         <y>40</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QUANTITE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="imprimer">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Imprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="imprimer_achat">
       <property name="geometry">
        <rect>
         <x>548</x>
         <y>140</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="chercher_id">
       <property name="geometry">
        <rect>
         <x>536</x>
         <y>260</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>539</x>
         <y>230</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Chercher</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Id_Fournisseur</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>186</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantit√©</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>250</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Date d'achat</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajouter_prix">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajouter_quantite">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>180</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>280</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="ajouter_date">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>240</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="ajouter_id">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Supprimer</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>115</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_FOURNISSEUR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>180</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QComboBox" name="supprimer_id">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>110</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Modifier</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>45</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_Fournisseur</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>112</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>175</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantit√©</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>245</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Date d'achat</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="modifier_prix">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="modifier_quantite">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>169</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="modifier_date">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>241</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>282</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="modifier_id">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>41</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Statistiques</string>
      </attribute>
      <widget class="QWidget" name="stat" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>621</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Vente</string>
    </attribute>
    <widget class="QTabWidget" name="tabvente">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>361</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tableView_vente">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>511</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Trier">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="triV">
       <property name="geometry">
        <rect>
         <x>524</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>MATRICULE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QUANTITE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="imprimerV">
       <property name="geometry">
        <rect>
         <x>552</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Imprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="imprimer_vente">
       <property name="geometry">
        <rect>
         <x>566</x>
         <y>121</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="chercher_matricule">
       <property name="geometry">
        <rect>
         <x>556</x>
         <y>250</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>220</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Chercher</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule Client</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantite</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>192</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>255</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Vente</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ajouter_matricule">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajouter_quantiteV">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>120</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajouter_prixV">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>190</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="ajouter_dateV">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>261</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Ajouter">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>282</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Supprimer</string>
      </attribute>
      <widget class="QComboBox" name="supprimer_matricule">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>111</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>172</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule Client</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Modifier</string>
      </attribute>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule Client</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantite</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>195</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Vente</string>
       </property>
      </widget>
      <widget class="QComboBox" name="modifier_matricule">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="modifier_quantiteV">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>120</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="modifier_prixV">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>190</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="modifier_dateV">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>260</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Modifier">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>280</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
